### DISCUSSION TITLE
4090 support

### DISCUSSION THREAD

**Discussion by LeeTsinghua** [pyg-team/pytorch_geometric#6259] [Q&A]
Hello pyg-team, I have just use 4090 with NGC container to run some pytorch program. Is there any schedule that th torch_geometric support cuda 11.8? OR is there any methods that help me to use torch_geometric in nvcr.io/nvidia/pytorch:22.11-py3?

**Comment by rusty1s** [comment]
I think CUDA 11.8 should already be supported when manually compiling from source. Would that be an option?
